.gitignore

```diff
@@ -6,3 +6,7 @@ classes
 /pod-babashka-buddy
 /pod-babashka-buddy.jar
 /pod-babashka-buddy.build_artifacts.txt
+.clj-kondo
+.lsp/.cache
```

Comment:
I think you could just add

```
.cache
```
and this would be sufficient for both clojure-lsp and clj-kondo.

Reply:
Thanks for the info 👍